From 4541f581846a1d68bdd8637f019f4eac14ae33ff Mon Sep 17 00:00:00 2001
From: Jiyong Park <jiyong@google.com>
Date: Thu, 29 Jun 2017 15:48:22 +0900
Subject: [PATCH 12/13] mido: Build camera.* with BOARD_VNDK_VERSION

Instead of relying on the global include paths, use the header
libraries. Global include paths are not available when building with
BOARD_VNDK_VERSION. pthread.h is included explicitly since it isn't
automatically available then.

Bug: 37342627
Test: BOARD_VNDK_VERSION=current m -j camera.msm8996

Change-Id: I210629684ff3451512e2dc4391f6718d5ebdd19b
---
 camera/QCamera2/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/camera/QCamera2/Android.mk b/camera/QCamera2/Android.mk
index 7031f61..1abcb20 100644
--- a/camera/QCamera2/Android.mk
+++ b/camera/QCamera2/Android.mk
@@ -134,6 +134,7 @@ endif
 ifeq ($(TARGET_TS_MAKEUP),true)
 LOCAL_SHARED_LIBRARIES += libts_face_beautify_hal libts_detected_face_hal
 endif
+LOCAL_HEADER_LIBRARIES := media_plugin_headers
 
 LOCAL_STATIC_LIBRARIES := android.hardware.camera.common@1.0-helper
 
-- 
2.7.4

